### DISCUSSION TITLE
Does OrganicMaps use any folders/files from a preciously installed Maps.me? i.e. is it safe to delete /MapsWithMe folder?

### DISCUSSION THREAD

**Discussion by sunwapta** [organicmaps/organicmaps#6728] [Q&A] [answered]
After migrating from Maps.me to OrganicMaps, I uninstalled Maps.me - but it left a bunch of files in the /MapsWithMe folder (including old downloaded maps). Does OrganicMaps use  anything in the /MapsWithMe folder, or is it safe to delete that entire folder?

**Comment by biodranik** [comment, marked as answer]
Backup your bookmarks and delete it.
